layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "shaderRoundingModeRTEFloat64": 0,
+        "shaderRoundingModeRTZFloat16": 0,
+        "shaderRoundingModeRTZFloat32": 0,
+        "shaderRoundingModeRTZFloat64": 0
+    },
+    "VkPhysicalDeviceMaintenance3Properties": {
+        "maxPerSetDescriptors": 1024,
+        "maxMemoryAllocationSize" : 536870912
+    },
+    "VkPhysicalDeviceMultiviewProperties": {
+        "maxMultiviewViewCount": 6,
+        "maxMultiviewInstanceIndex" : 67108863
+    },
+    "VkPhysicalDeviceProtectedMemoryProperties": {
+        "protectedNoFault": 0
+    },
+    "VkPhysicalDeviceSamplerFilterMinmaxProperties": {
+        "filterMinmaxSingleComponentFormats": 0,
+        "filterMinmaxImageComponentMapping": 0
+    },
+    "VkPhysicalDeviceTimelineSemaphoreProperties": {
+        "maxTimelineSemaphoreValueDifference": 1073741823
+    },
+    "VkPhysicalDeviceFeatures": {
+        "robustBufferAccess": 1,
+        "fullDrawIndexUint32": 0,
+        "imageCubeArray": 0,
+        "independentBlend": 0,
+        "geometryShader": 0,
+        "tessellationShader": 0,
+        "sampleRateShading": 0,
+        "dualSrcBlend": 0,
+        "logicOp": 0,
+        "multiDrawIndirect": 0,
+        "drawIndirectFirstInstance": 0,
+        "depthClamp": 0,
+        "depthBiasClamp": 0,
+        "fillModeNonSolid": 0,
+        "depthBounds": 0,
+        "wideLines": 0,
+        "largePoints": 0,
+        "alphaToOne": 0,
+        "multiViewport": 0,
+        "samplerAnisotropy": 0,
+        "textureCompressionETC2": 0,
+        "textureCompressionASTC_LDR": 0,
+        "textureCompressionBC": 0,
+        "occlusionQueryPrecise": 0,
+        "pipelineStatisticsQuery": 0,
+        "vertexPipelineStoresAndAtomics": 0,
+        "fragmentStoresAndAtomics": 0,
+        "shaderTessellationAndGeometryPointSize": 0,
+        "shaderImageGatherExtended": 0,
+        "shaderStorageImageExtendedFormats": 0,
+        "shaderStorageImageMultisample": 0,
+        "shaderStorageImageReadWithoutFormat": 0,
+        "shaderStorageImageWriteWithoutFormat": 0,
+        "shaderUniformBufferArrayDynamicIndexing": 0,
+        "shaderSampledImageArrayDynamicIndexing": 0,
+        "shaderStorageBufferArrayDynamicIndexing": 0,
+        "shaderStorageImageArrayDynamicIndexing": 0,
+        "shaderClipDistance": 0,
+        "shaderCullDistance": 0,
+        "shaderFloat64": 0,
+        "shaderInt64": 0,
+        "shaderInt16": 0,
+        "shaderResourceResidency": 0,
+        "shaderResourceMinLod": 0,
+        "sparseBinding": 0,
+        "sparseResidencyBuffer": 0,
+        "sparseResidencyImage2D": 0,
+        "sparseResidencyImage3D": 0,
+        "sparseResidency2Samples": 0,
+        "sparseResidency4Samples": 0,
+        "sparseResidency8Samples": 0,
+        "sparseResidency16Samples": 0,
+        "sparseResidencyAliased": 0,
+        "variableMultisampleRate": 0,
+        "inheritedQueries": 0
+    },
+    "VkPhysicalDevice16BitStorageFeatures": {
+        "storageBuffer16BitAccess": 1,
```

Comment:
This one is more strange:
Promotion to Vulkan 1.1
```All functionality in this extension is included in core Vulkan 1.1, with the KHR suffix omitted. However, if Vulkan 1.1 is supported and this extension is not, the storageBuffer16BitAccess capability is optional. The original type, enum and command names are still available as aliases of the core functionality.```

So this is optional, also it's supported by most (all?) desktop drivers.

Reply:
Good catch.
I'll look into it and probably set it to "Not supported" instead of "Supported".